client/src/App.jsx

```diff
@@ -10,6 +11,7 @@ function App() {
       <Router>
         <Routes>
           <Route path='/' element={<Home />} />
+          <Route path='/create-account' element={<CreateAccount/>}></Route>
```

Comment:
You should use a self-closing tag here, like so:

```
<Route path='/create-account' element={<CreateAccount/>} />
```

This is a little easier to read, and is consistent with the previous line.